packages/dev/docs/pages/releases/2023-10-06.mdx

```diff
@@ -0,0 +1,102 @@
+
+As always, huge thanks to everyone in our community for their feedback and contributions to make this release possible!
+
+### FileTrigger API update
+
+This release of FileTrigger includes an API change. If you were using a previous version, you will need to update your code. See below for details.
+- The `OnChange` prop has been replaced with `OnSelect`.
+
+## Enhancements
+
+- Add staticColor prop to ProgressCircle and ProgressBar - [@iamwillpowell](https://github.com/iamwillpowell) - [PR](https://github.com/adobe/react-spectrum/pull/5108)
+- Add support for 'none' as a BorderSizeValue to hide border - [@pr7prashant](https://github.com/pr7prashant) - [PR](https://github.com/adobe/react-spectrum/pull/5012)
+- Support links in collection components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4993)
+- Cover all plural forms in NumberParser - [@jbovenschen](https://github.com/jbovenschen) - [PR](https://github.com/adobe/react-spectrum/pull/5134)
+- Add TypeScript StrictMode to React Stately utils - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3926)
+
+## Fixes
+
+- Update Toast placement to bottom center - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5107)
+- Fix Unavailable MenuItem props - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5153)
+- Fix Popover placement for placement="start"|"end" - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/5137)
+- Remove Express icon dependency - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5130)
+- Fix NumberField form submission - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4300)
+- Update Express Breadcrumb icon - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5059)
+- Fix client navigation check in Firefox - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5106)
+- Fix Breadcrumbs menu padding - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5086)
```

Comment:
this fell into links/and the other breadcrumb PR, no need to include it

the general gist is people can come here and see what changed for the version we're releasing, so we should be able to quickly look through this list for overall thematic changes